.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup PHP and Composer
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '5.6'
+          php-version: '7.3'
```

Comment:
Is there a specific reason to use 7.3 instead of the latest stable 7.4? See https://www.php.net/supported-versions.php

Reply:
That was the minimum that PHPUnit 9.5 required, but 7.4 sounds good to me too, I'll update it to that 👍🏻

Reply:
Updated in [8cbcd57](https://github.com/WordPress/two-factor/pull/472/commits/8cbcd579e4c26fe789099c398de2f1225049048a). Looks like the checks are still passing 👍🏻